tests/test_benchmark/test_gritlm_models.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict, List, Union
+
+import pytest
+import torch
+
+import mteb
+from mteb.models.instructions import task_to_instruction
+from tests.test_benchmark.task_grid import MOCK_TASK_TEST_GRID
+
+pytest.importorskip("gritlm")
+from gritlm import GritLM  # noqa
+
+
+class MockGritLMWrapper(GritLM):
```

Comment:
Not entirely sure what this tests in the original implementation? Isn't it simply reimplementing the functionality?

Reply:
Yes, a bit. All GritLM wrappers covered inside functions, so I can't just inherit them, but I think test that GritLM models can run tests could be helpful

Reply:
Hmm, that might be better done by creating an extremely small grim model (>100m parameters) and just testing that instead. Feel free to do that; otherwise, I would remove it for now and leave it for another PR.

Reply:
Tried to run gritlm with `sentence-transformers/all-MiniLM-L6-v2`, but got error, that passed unexpected parameters by gritlm library. Removed this test